core-bundle/src/Altcha/Config/AlgorithmConfig.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Altcha\Config;
+
+enum AlgorithmConfig: string
```

Comment:
Why `Config`? This is an enum so it can only represent one value. It is an `Aglorithm`, not a config :) So imho you can just drop the `Config` suffix here.

Just 

```php
enum Algorithm: string
{
    case SHA_256 = 'SHA-256';
    case SHA_384 = 'SHA-384';
    case SHA_512 = 'SHA-512';
    
   public static function values(): array
   {
        return array_column(self::cases(), 'value');
   }
}


var_dump(Algorithm::values());
```

Then you also have your `array_column()` logic in a central place :)